clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaConsumerService.java

```diff
@@ -118,7 +118,7 @@ protected KafkaConsumerService(
       /**
        * We need to assign a unique client id across all the storage nodes, otherwise, they will fail into the same throttling bucket.
        */
-      consumerProperties.setProperty(ConsumerConfig.CLIENT_ID_CONFIG, getUniqueClientId(kafkaUrl, i));
+      consumerProperties.setProperty(KAFKA_CLIENT_ID_CONFIG, getUniqueClientId(kafkaUrl, i));
```

Comment:
Why is this better? CLIENT_ID_CONFIG seems more generic?

Reply:
As we want to move all the kafka dependency from our code to Apache Kafka Adapters, the `CLIENT_ID_CONFIG` is from kafka client package, replacing it with  `KAFKA_CLIENT_ID_CONFIG` from the `ConfigKeys`, and `ApacheKafkaAdminConfig` will translate it into `CLIENT_ID_CONFIG`  for constructing consumers.